internal/config/config_test.go

```diff
@@ -60,6 +60,7 @@ func TestSetup(t *testing.T) {
 }
 
 func TestWatchedFoldersConfig(t *testing.T) {
+	os.Setenv("GDG_CONTEXT_NAME", "qa")
```

Comment:
I think this line does nothing given the following block? Is this leftover testing?